lib/seadsa/DsaLocal.cc

```diff
@@ -1181,6 +1198,12 @@ void IntraBlockBuilder::visitSeaDsaFnCall(CallSite &CS) {
     return;
   }
 
+  case SeadsaFn::NEW: {
+    //generate an allocation call with no attributes set
+    visitAllocationFnCall(CS, false);
```

Comment:
I imagined that `new` allocates memory on the heap. I would suggest to use `dsa_alloca` for allocating memory on the stack. If the memory is allocated on the stack, it should not flow outside of the function. So a strange thing to have in the specification language ...